data/src/main/java/me/androidbox/data/local/event/EventRepositoryImp.kt

```diff
@@ -51,6 +53,7 @@ class EventRepositoryImp @Inject constructor(
 
         val result = checkResult {
             eventDao.insertEvent(eventEntity)
+            eventDao.insertSyncEvent(EventSyncEntity(event.id, SyncAgendaType.CREATE))
```

Comment:
@MKiperszmid @philipplackner Is it ok to run these one after another.

Reply:
You could use a transaction to only execute both if both succeed. You can do this by using `db.withTransaction`. Very unlikely inserting into a DB fails, but theoretically possible